test/Microsoft.IdentityModel.Logging.Tests/project.json

```diff
@@ -1,14 +1,28 @@
 ﻿{
   "version": "1.0.0-*",
   "dependencies": {
-    "Microsoft.IdentityModel.Tokens.Tests": "5.0.0-*"
+    "Microsoft.IdentityModel.Tokens.Tests": "5.0.0-*",
+    "xunit": "2.1.0"
   },
   "testRunner": "xunit",
   "frameworks": {
     "netstandardapp1.5": {
-      "imports": [ "dnxcore50", "portable-net451+win8" ]
+      "dependencies": {
+        "dotnet-test-xunit": "1.0.0-dev-*",
```

Comment:
We don't need to include these dependencies here, they gets pulled in by Microsoft.IdentityModel.Tokens.Tests. I took them off in a recent PR. Was it breaking without this?